src/install/config/pasta.yaml

```diff
@@ -0,0 +1,36 @@
+agentName: java
```

Comment:
just at a quick glance i assume that this is the issue! since there's already an agent with the name `java`

Reply:
I believe I had it called `pasta` until I tried `java` to troubleshoot. But let me double check in case I'm wrong!

Reply:
Yeah still 404ing locally w/out an error even with the new commit :(

Reply:
so strange - i'm not having an issue building it locally 
<img width="1179" alt="Screen Shot 2022-09-09 at 7 39 49 PM" src="https://user-images.githubusercontent.com/38332422/189459212-7ea28d75-571c-4eb6-b9c3-844831bf0ed6.png">